.github/workflows/firebase-merge.yml

```diff
@@ -1,6 +1,12 @@
 name: Deploy to Firebase on merge
 on:
   push:
+    paths:
```

Comment:
Should we include `database.rules.json`?

Reply:
Good catch, done.